scheme-libs/racket/unison/.gitignore

```diff
@@ -0,0 +1,6 @@
+compiled/
+boot-generated.ss
+builtin-generated.ss
+compound-wrappers.ss
+data-info.ss
+simple-wrappers.ss
```

Comment:
I _think_ this is okay.
@dolio is this okay?

Reply:
Yeah, pretty sure it's fine to ignore these. The .ss files are auto-generated from unison, and `compiled/` is racket's build directory.